templates/zerver/help/format-your-message-using-markdown.md

```diff
@@ -72,17 +72,17 @@ A complete list of emojis can be found [here](http://www.webpagefx.com/tools/emo
 
 ## Mentions
 
-To call the attention of another member, you can alert other users by typing `@**username**"`
+To call the attention of another member, you can alert them by typing `@**username**`
```

Comment:
Hm I thought "them" would be good here. Im not native though. Can anyone confirm "them" is incorrect?